protos/transponder/transponder.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package mavsdk.rpc.transponder;
+
+import "mavsdk_options.proto";
+
+option java_package = "io.mavsdk.transponder";
+option java_outer_classname = "TransponderProto";
+
+/*
+ * Allow users to get ADS-B information
+ * and set ADS-B update rates.
+ */
+service TransponderService {
+    // Subscribe to 'transponder' updates.
+    rpc SubscribeTransponder(SubscribeTransponderRequest) returns(stream TransponderResponse) {}
+    
+    // Set rate to 'transponder' updates.
+    rpc SetRateTransponder(SetRateTransponderRequest) returns(SetRateTrasponderResponse) {}
+    
+}
+
+message SubscribeTransponderRequest {}
+message TransponderResponse {
+    ADSB_VEHICLE transponder = 1; // The next detection 
```

Comment:
What's `ADSB_VEHICLE`? Where is that defined? Also shouldn't it be called `AdsbVehicle`?

Reply:
I'm assuming it should be a struct similar to [this](https://mavlink.io/en/messages/common.html#ADSB_VEHICLE). I agree though the case and spacing I have is not appropriate for this file as I believe it requires camel case.

Reply:
You need to define it yourself, like `telemetry` does for `Position` here: https://github.com/mavlink/MAVSDK-Proto/blob/main/protos/telemetry/telemetry.proto#L451-L457

And then in the C++ implementation, you'll set the fields of your `AdsbVehicle` struct with the ones coming from mavlink, a bit like [here](https://github.com/mavlink/MAVSDK/blob/main/src/plugins/telemetry/telemetry_impl.cpp#L604-L614)

Reply:
Great thanks!